tests/foreman/ui/test_templatesync.py

```diff
@@ -76,48 +81,55 @@ def test_positive_import_templates(session, templates_org, templates_loc):
         assert pt['template']['name'] == imported_template
         assert pt['template']['default'] is False
         assert pt['type']['snippet'] is False
+        assert pt['template']['locked'] is True
         assert pt['locations']['resources']['assigned'][0] == templates_loc.name
         assert pt['organizations']['resources']['assigned'][0] == templates_org.name
         assert f'name: {import_template}' in pt['template']['template_editor']['editor']
 
 
 @tier2
 @upgrade
-def test_positive_export_templates(session):
+def test_positive_export_templates(session, create_import_export_local_dir):
```

Comment:
```suggestion
def test_positive_export_templates(session):
```
forgot to mention this one.

Reply:
Resolved!